### PR TITLE
 #161254628 Implement social sharing

### PR DESCRIPTION
**Pull request by evamaina**
#### What does this PR do?
```This PR implements social sharing via Facebook, Email, Twitter and Facebook ```
#### Description of Task to be completed?
```A user should be able to share/post/tweet articles on social media platforms like Facebook, Email, WhatsApp and Twitter ```
#### How should this be manually tested?
- Clone this repository
```git clone https://github.com/andela/ah-orcas-frontend.git```
- Change the working directory to the project folder
  `cd ah-orcas-frontend`
- Checkout to the appropriate branch
  `git checkout ft-social-sharing-161254628`
- Install the required dependencies:
   `npm start`
- You should be able to see the social buttons beside every article. Click on the one you want to use to share the article.
#### What are the relevant pivotal tracker stories?
[#161254628](https://www.pivotaltracker.com/story/show/161254628)
#### Screenshots
image

#### Checklist:

- [x] My code follows the style guidelines of this project
- [ ] At least 2 people have reviewed my PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

### CODE REVIEW

**Comment by Eyansky** [comment]
Your commit is not signed.

File: src/components/SocialShare/ShareComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        </WhatsappShareButton>
+        <TwitterShareButton
+          url={`http://ah-orcas-frontend-staging.herokuapp.com/article/${slug}`}
+          title={title}
+          className="share-icon"
+        >
+          <TwitterIcon size={32} round />
+        </TwitterShareButton>
+        <FacebookShareButton
+          url={`http://ah-orcas-frontend-staging.herokuapp.com/article/${slug}`}
```
Comment (TheSteelGuy) [inline src/components/SocialShare/ShareComponent.jsx]: this one too and the rest

File: src/components/SocialShare/ShareComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+class SocialShare extends Component {
+  componentDidMount() {}
+
+  render() {
+    const { title, slug } = this.props;
+    return (
+      <div>
+        <WhatsappShareButton
+          url={`http://ah-orcas-frontend-staging.herokuapp.com/article/${slug}`}
```
Comment (TheSteelGuy) [inline src/components/SocialShare/ShareComponent.jsx]: Get this URL from env variable

**Review by TheSteelGuy — CHANGES_REQUESTED** [review]

**Comment by evamaina** [comment]
> Your commit is not signed.

It's configured but not working I don't know why.

File: src/components/SocialShare/ShareComponent.jsx (hunk 2 above)
Comment (evamaina) [reply src/components/SocialShare/ShareComponent.jsx]: oops, I forgot. on it

**Review by TheSteelGuy — APPROVED** [review]

**Comment by Eyansky** [comment]
LGTM

**Review by Eyansky — APPROVED** [review]

**Comment by abadojack** [comment]
LGTM

**Review by wrotich — APPROVED** [review]
LGTM

**Pull request merged**